src/renderers/dom/test/__tests__/ReactTestUtils-test.js

```diff
@@ -259,8 +259,34 @@ describe('ReactTestUtils', () => {
     );
 
     expect(() => ReactTestUtils.Simulate.click(result)).toThrowError(
-      'TestUtils.Simulate expects a component instance and not a ReactElement.' +
-        'TestUtils.Simulate will not work if you are using shallow rendering.',
+      'TestUtils.Simulate expected a DOM node as the first argument but received ' +
+        'a React element. Pass the DOM node you wish to simulate the event on instead. ' +
+        'Note that TestUtils.Simulate will not work if you are using shallow rendering.',
+    );
+    expect(handler).not.toHaveBeenCalled();
+  });
+
+  it('should throw when attempting to use a component instance', () => {
+    class SomeComponent extends React.Component {
```

Comment:
Same issue: the test name is vague ("use" how?)

Let's change to "should throw if Simulate receives a component instance".

Also we should probably group both tests under existing Similate describe block.